js/index.js

```diff
@@ -1,9 +1,353 @@
-function getCurrentTime() {
-    const now = new Date();
-    const hours = now.getHours().toString().padStart(2, '0');
-    const minutes = now.getMinutes().toString().padStart(2, '0');
-    const seconds = now.getSeconds().toString().padStart(2, '0');
-    return `${hours}:${minutes}:${seconds}`;
-}
-
-console.log(getCurrentTime());
\ No newline at end of file
+'use strict'
+{
+
+  const sysmember = [
+    { "src": "./images/otakesan.webp", "nickname": "おたけ", "value": "otake" },
+    { "src": "./images/riri-san.webp", "nickname": "リリー", "value": "riri-" },
+    { "src": "./images/mossan.webp", "nickname": "もっさん", "value": "mossan" },
+    { "src": "./images/shimopi-san.webp", "nickname": "しもぴー", "value": "shimopi-" },
+    { "src": "./images/oga00385.jpg", "nickname": "ななみん", "value": "nanamin" },
+    { "src": "./images/makottyan.webp", "nickname": "まこっちゃん", "value": "makottyan" },
+    { "src": "./images/utti-san.webp", "nickname": "うっちー", "value": "utti-" },
+    { "src": "./images/akuisan.webp", "nickname": "だいちゃん", "value": "daityan" },
+    { "src": "./images/yoshiokasan.webp", "nickname": "吉岡さん", "value": "yoshiokasan" },
+    { "src": "./images/nagayansan.webp", "nickname": "ながやん", "value": "nagayan" },
+    { "src": "./images/oga00331.webp", "nickname": "みっぴー", "value": "mippi-" },
+  ];
+  //ニックネームを回答するモード
+  let nicknameMode = true;
+  //顔を回答するモード
+  let faceMode = false;
+  //表示インデックス
+  let randomIndex;
+  //問題数を格納 最終問題かの判定を行う
+  const questionLength = sysmember.length;
```

Comment:
下記のような書き方もありかな。
クイズの上限の仕様が把握できそう。
```suggestion
  // 全メンバー分を出題したらクイズ終了となるため、メンバー数を取得し最大の問題数にセットする
  const maxQuestionNum = sysmember.length;
```